### PR TITLE
Built-in functions and global scope doc

### PR DESCRIPTION
**Pull request by martindzejky**
I added descriptions for built-in functions (more verbose ones) and some global scope constants.

### CODE REVIEW

File: doc/base/classes.xml [hunk 1]
```diff
@@ -772,119 +775,172 @@
...
+			Insert Key
 		</constant>
 		<constant name="KEY_DELETE" value="16777224">
+			Delete Key
 		</constant>
 		<constant name="KEY_PAUSE" value="16777225">
+			Pause Key
 		</constant>
 		<constant name="KEY_PRINT" value="16777226">
+			Printscreen Key
```
Comment (chuckeles) [inline doc/base/classes.xml]: Couldn't get this working in Godot, is it correct?

File: doc/base/classes.xml [hunk 2]
```diff
@@ -1384,36 +1480,52 @@
...
+			Joystick Right Trigger 1
 		</constant>
 		<constant name="JOY_R2" value="7">
+			Joystick Right Trigger 2
 		</constant>
 		<constant name="JOY_R3" value="9">
+			Joystick Right Trigger 3
 		</constant>
 		<constant name="JOY_AXIS_0" value="0">
+			Joystick Left Pad Horizontal Axis
```
Comment (chuckeles) [inline doc/base/classes.xml]: Couldn't test this, are these axis correct?

**Comment by Hinsbart** [comment]
The joystick descriptions are correct, yes. But I would choose the wording a bit differently.
For `JOY_AXIS_*`, I'd say "stick" instead of "Pad". 
For `JOY_ANALOG_*` I'd use the same as for `JOY_AXIS_*` and scrap the "Analog" part, as axes should always be analog. I guess the point of these constants is that the user can instantly see if it's X or Y.
`JOY_L/R`: "Joystick left/right shoulder button"
`JOY_L2/R2`: "Joystick left/right trigger"
`JOY_L3/R3`: "Joystick left/right stick click"

**Comment by Hinsbart** [comment]
Also, `JOY_AXIS_4/5`: "Joystick left/right trigger axis" or maybe "analog" would be appropriate here since you can get the triggers both digitally as buttons and analog as axes.

**Comment by chuckeles** [comment]
Thanks, I'll rewrite those descriptions. And squash to one commit.

**Comment by chuckeles** [comment]
And done :smiley:

**Comment by Hinsbart** [comment]
oops, seems like I wasn't thinking so straight :/
The triggers should be axes 6 and 7, sorry.

fixed by #3949

**Pull request merged**